cmd/update.go

```diff
@@ -76,12 +76,12 @@ var updateChunkNumber int
 func init() {
 	rootCmd.AddCommand(updateCmd)
 	updateCmd.PersistentFlags().String("allocation", "", "Allocation ID")
-	updateCmd.PersistentFlags().String("remotepath", "", "Remote path to upload")
-	updateCmd.PersistentFlags().String("localpath", "", "Local path of file to upload")
-	updateCmd.PersistentFlags().String("thumbnailpath", "", "Local thumbnail path of file to upload")
-	updateCmd.Flags().Bool("encrypt", false, "(default false) pass this option to encrypt and upload the file")
+	updateCmd.PersistentFlags().String("remotepath", "", "Remote path to update")
```

Comment:
It should be "Remote path of file to update"

Reply:
done